content/blog/2025/oddcontrast-guide.md

```diff
@@ -0,0 +1,130 @@
+---
+title: OddContrast Title
+sub:: A guide to using OddContrast
+author: oddbird
+date: 2025-05-20
+tags:
+  - Article
+  - Accessibility
+  - Color
+  - Design
+  - OddTools
+image:
+  src: blog/2025/oddcontrast-gamut-sliders.png
+  alt: >
+    OddContrast displays sRGB gamut range
+    in P3 color format and an estimate
+    of foreground alpha value ratio.
+summary: |
+  Learn how to use OddBird's color tool that not only checks contrast ratios
+  but also utilizes the color formats and spaces newly available in CSS.
+---
+
+## Background we might not need?
```

Comment:
I had that larger chunk of text at the bottom here, but I think this is probably a short enough intro without getting too technical into the details of each color space. Needs a better heading here :)